src/month.js

```diff
@@ -90,6 +105,24 @@ export default function CalendarMonth(props) {
               {
                 days.map((day) => (
                   <div key={day.format('YYYYMMDD')} className="tt-cal-day">
+                    { (
```

Comment:
And in response to the nit, logic being more than a few lines removed from where it actually applies sometimes hurts readability for me, too. I think you're probably right in this case, though.